Sources/AdminPanel/Controllers/AdminPanelUserController.swift

```diff
@@ -23,13 +24,18 @@ public final class AdminPanelUserController
 
     public func renderList(_ req: Request) throws -> Future<Response> {
         let config: AdminPanelConfig<U> = try req.make()
-        return U.query(on: req).all()
-            .flatMap(to: Response.self) { users in
+        let paginator: Future<OffsetPaginator<U>> = try U.query(on: req).paginate(for: req)
```

Comment:
Is it still necessary to specify the explicit type here?

Reply:
I'm afraid so since the return type is `Response`.